File: server/FlightLog-server/src/main/java/bitc/fullstack503/flightlog/flightlogserver/dto/iFlightDTO.java
package bitc.fullstack503.flightlog.flightlogserver.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class iFlightDTO {

  @JsonProperty("airlineKorean")
  private String airlineKorean;

  @JsonProperty("arrvcity")
  private String arrvcity;

  @JsonProperty("internationalEddate")
  private String internationalEddate;

  @JsonProperty("internationalMon")
  private String internationalMon;

  @JsonProperty("internationalTue")
  private String internationalTue;

  @JsonProperty("internationalWed")
  private String internationalWed;

  @JsonProperty("internationalThu")
  private String internationalThu;

  @JsonProperty("internationalFri")
  private String internationalFri;

  @JsonProperty("internationalSat")
  private String internationalSat;

  @JsonProperty("internationalSun")
  private String internationalSun;

  @JsonProperty("internationalNum")
  private String internationalNum;

  @JsonProperty("internationalStdate")
  private String internationalStdate;

  @JsonProperty("internationalTime")
  private String internationalTime;

  @JsonProperty("deptcity")
  private String deptcity;



}

File: app/app/src/main/java/bitc/fullstack/FlightLog/dto/iFlightDTO.kt
package bitc.fullstack.FlightLog.dto

import com.google.gson.annotations.SerializedName

// 서버로 전송하는 국내 항공 정보
data class iFlightDTO(

    @SerializedName("airlineKorean")
    var airlineKorean: String,

    @SerializedName("arrvcity")
    var arrvcity: String,

    @SerializedName("internationalEddate")
    var internationalEddate: String,

    @SerializedName("internationalMon")
    var internationalMon: String,

    @SerializedName("internationalTue")
    var internationalTue: String,

    @SerializedName("internationalWed")
    var internationalWed: String,

    @SerializedName("internationalThu")
    var internationalThu: String,

    @SerializedName("internationalFri")
    var internationalFri: String,

    @SerializedName("internationalSat")
    var internationalSat: String,

    @SerializedName("internationalSun")
    var internationalSun: String,

    @SerializedName("internationalNum")
    var internationalNum: String,

    @SerializedName("internationalStdate")
    var internationalStdate: String,

    @SerializedName("internationalTime")
    var internationalTime: String,

    @SerializedName("deptcity")
    var deptcity: String
)
